src/command/vanilla/version-command.js

```diff
@@ -0,0 +1,28 @@
+/*
+This command is non-vanilla, but exists in PMMP, Altay, MiNET, Nukkit a.s.o.
+*/
+
+const Command = require('../command')
+const Player  = require('../../player')
+const ConsoleSender = require('../console-sender')
+
+class VersionCommand extends Command {
+
+    constructor() {
+        super({ name: 'version', description: 'Displays general server informations.', aliases: ['about']})
+    }
+
+    /**
+     * @param {ConsoleSender|Player} sender 
+     * @param {Array} args 
+     */
+    execute(sender, args) {
+        let server_version    = require('../../../package.json').version               //The only version reference I found, please contribute if there is a "better" one
+        let protocol_version  = require('../../network/identifiers').Protocol
```

Comment:
Couldn't you just use one import here, rather than being spammy with `require().version`?

Reply:
Yes, I could. I didn't rather that as too spammy.